### PR TITLE
remove the statsd in multi cluster setup

### PR DESCRIPTION
**Pull request by incfly**
We've been deprecated statsd according to values.yaml. We should not mention it in the multi cluster documentation.
https://github.com/istio/istio/issues/11683

### CODE REVIEW

**Comment by incfly** [comment]
@mandarjog @sdake PTAL

**Comment by linsun** [comment]
can you pls confirm if these are the only places that need to be updated?

**Comment by incfly** [comment]
@douglas-reid I found the mentioning statds in should be deprecated was in your PR https://github.com/istio/istio/pull/8485

I'm updating multi cluster documentation and then find this `statsd` in the doc is outdated. And do a larger scale sweep. 

Can you take a look and ensure this is safe? Thanks!

**Comment by geeknoid** [comment]
To fix the CI failure, please rebase and push again. Sorry about that.

**Comment by incfly** [comment]
/assign @gyliu513 
for `content_zh/OWENRS`

File: content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md [hunk 1]
```diff
@@ -54,8 +54,7 @@ aliases:
 
 等待 Istio 控制平面完成初始化，然后再执行本节中的步骤。
 
-你必须在 Istio 控制平面上运行这些操作以捕获 Istio 控制平面服务端点，如：Pilot、Policy 和
-Statsd Pod IP 端点。
+你必须在 Istio 控制平面上运行这些操作以捕获 Istio 控制平面服务端点，如：Pilot、Policy端点。
```
Comment (rootsongjc) [inline content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md]: 中文文档中要求中文和英文之间要加空格。

File: content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md [hunk 2]
```diff
@@ -376,15 +370,14 @@ pod 重启的一个简单的解决方案是给 Istio 服务使用负载均衡器
...
-* `istio-statsd-prom-bridge`
 * `zipkin`
 
 目前，Istio 安装不提供为 Istio 服务指定服务类型的选项。 您可以在 Istio Helm chart 或 Istio 清单中手动指定服务类型。
 
 ### 通过网关暴露 Istio 服务
 
 此方法使用了 Istio ingress 网关的功能。远程集群有 `istio-pilot`、`istio-telemetry`、 `istio-policy`、
-`istio-statsd-prom-bridge` 和 `zipkin` 服务，它们指向了 Istio ingress 网关的负载均衡 IP 地址。然后，所有的服务均指向相同的IP 地址。你必须创建目标规则以在主集群的 ingress 网关中获得正确的 Istio 服务。
+和 `zipkin` 服务，它们指向了 Istio ingress 网关的负载均衡 IP 地址。然后，所有的服务均指向相同的IP 地址。你必须创建目标规则以在主集群的 ingress 网关中获得正确的 Istio 服务。
```
Comment (rootsongjc) [inline content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md]: 第二个 IP 前少一个空格。

Comment (incfly) [reply content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md]: Done.

File: content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md (hunk 1 above)
Comment (incfly) [reply content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md]: Done. 所以这是为什么？方便search engine做索引么?

Comment (rootsongjc) [reply content_zh/docs/setup/kubernetes/multicluster-install/vpn/index.md]: 方便分词是一个好处，另外是为了排版更美观。

File: content/docs/examples/multicluster/gke/index.md [hunk 3]
```diff
@@ -148,9 +147,7 @@ $ kubectl get pods -n istio-system
       --name istio-remote \
       --set global.remotePilotAddress=${PILOT_POD_IP} \
```
Comment (sdake) [inline content/docs/examples/multicluster/gke/index.md]: for followup work, you also shouldn't need `remotePolicyAddress` nor `remoteTelemetryAddress`, although you would of course want to test that :)

File: content/blog/2018/incremental-traffic-management/index.md [hunk 4]
```diff
@@ -67,7 +67,6 @@ $ helm template install/kubernetes/helm/istio \
   --set sidecarInjectorWebhook.enabled=false \
```
Comment (sdake) [inline content/blog/2018/incremental-traffic-management/index.md]: note the istio-remote chart has been deprecated and merged into the istio chart.  We should probably just remove istio-remote entirely from the repository as it is confusing and doesn't work as is.  I am struggling to find the exact flag - @ymesika knows - I believe he was doing this consolidation worrk.

File: content/docs/reference/config/installation-options/index.md [hunk 5]
```diff
@@ -156,9 +156,6 @@ To customize Istio install using Helm, use the `--set <key>=<value>` option in H
 | `global.proxy.includeInboundPorts` | `"*"` |  |
```
Comment (sdake) [inline content/docs/reference/config/installation-options/index.md]: fwiw No need to modify this file - it is autogenerated.  But thanks anyway - we can autogenerate prior to 1.1 with the 1.1 binaries.  Cheers :)

File: content/docs/examples/multicluster/gke/index.md (hunk 3 above)
Comment (incfly) [reply content/docs/examples/multicluster/gke/index.md]: By follow up work, you mean the single gateway load balancer IP to expose the telemetry, Pilot service?

File: content/docs/reference/config/installation-options/index.md (hunk 5 above)
Comment (incfly) [reply content/docs/reference/config/installation-options/index.md]: Yeah, but I didn't find the make target of how to update this file.

**Pull request merged**